_posts/2019-09-23-hhvm-4.24.0.markdown

```diff
@@ -0,0 +1,56 @@
+---
+title: "HHVM 4.24.0"
+layout: post
+author: jjergus
+category: blog
+---
+
+HHVM 4.24 is released! This release marks the end of support for 4.18;
+4.19-4.23 remain supported, as do the LTS releases 3.30 and 4.8.
+
+# Highlights
+
+- We are now building packages for Debian 10 (Buster). Debian 10 support is currently experimental, please [open an issue](https://github.com/facebook/hhvm/issues/new) if you notice any problems.
+- Fixed a bug that could cause segfaults during JIT on Ubuntu 19.04 (Disco Dingo).
```

Comment:
maybe 'in JIT internals' or similar? I'd like it to be a bit clearer that it's not while executing code; probably too detailed for release notes, but it's when generating IR